client/components/SurrenderForm.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorsList"
+
+const SurrenderForm = (props) => {
+  const [message, setMessage] = useState("")
+  const [errors, setErrors] = useState({})
+  const [surrender, setSurrender] = useState({
+    personName: "",
+    phoneNumber: "",
+    email: "",
+    petName: "",
+    age: "",
+    petTypeId: "",
+    imgUrl: "",
+    vaccinationStatus: "",
+    adoptionStory: "",
+  })
+
+  const handleInputChange = (event) => {
+    setSurrender({
+      ...surrender,
+      [event.currentTarget.id]: event.currentTarget.value,
+    })
+  }
+
+  const submitHandler = (event) => {
+    event.preventDefault()
+    if (isValid()) {
+      addPetForSurrender()
+    } else {
+      console.log("Inputs not valid!")
+    }

```

Comment:
since `isValid()` is already adding error messages to `errors` that will be passed to the `ErrorList` component we can remove the `else` part of the condition